defaults/main.yml

```diff
@@ -31,12 +31,15 @@ casd_metrics_args: >-
 casd_bind_address: "0.0.0.0"
 casd_bind_path: ""
 casd_bind: "{{ 'unix:' ~ casd_bind_path if casd_bind_path else casd_bind_address ~ ':' ~ casd_port }}"
+# (debug, error, info, trace, warning) [optional, default = "error"]
+casd_loglevel: ""
 casd_quota_high: ""
 casd_reserved: ""
 casd_cache_mnt: "/srv"
 casd_certs_mnt: "/certs"
 casd_cmd: >-
   --verbose --bind {{ casd_bind }}
```

Comment:
@nanonyme should we remove the "--verbose" from here?

Reply:
Definitely. Looks like I messed up with web UI. Unless someone beats me to it, I will probably fix in the morning.

Reply:
The point is log level needs to be configurable without touching this repo.

Reply:
I removed the hardcoded `--verbose` flag.